requirements-app.txt

```
@@ -1,7 +1,7 @@
 # Run `make freeze-requirements` to update requirements.txt
 # with package version changes made in requirements-app.txt
 
-Flask==1.0.3
+Flask==1.0.3 # pyup: ignore
```

I would pin to a range rather than blanket ignoring

```suggestion
Flask==1.0.3 # pyup: >=1.0.0,<1.1.0
```